src/Responses/FineTuning/RetrieveJobResponseHyperparameters.php

```diff
@@ -8,29 +8,33 @@
 use OpenAI\Responses\Concerns\ArrayAccessible;
 
 /**
- * @implements ResponseContract<array{n_epochs: int|string}>
+ * @implements ResponseContract<array{n_epochs: int|string, batch_size: int|string, learning_rate_multiplier: int|string}>
  */
 final class RetrieveJobResponseHyperparameters implements ResponseContract
 {
     /**
-     * @use ArrayAccessible<array{n_epochs: int|string}>
+     * @use ArrayAccessible<array{n_epochs: int|string, batch_size: int|string, learning_rate_multiplier: int|string}>
      */
     use ArrayAccessible;
 
     private function __construct(
         public readonly int|string $nEpochs,
+        public readonly int|string $batchSize,
+        public readonly int|string $learningRateMultiplier,
```

Comment:
According to the docs this should be a float: https://platform.openai.com/docs/api-reference/fine-tuning/create
```suggestion
        public readonly float|string $learningRateMultiplier,
```

Reply:
ok on doc `learning_rate_multiplier` can be also null so setted `float|string|null`
I added also `prompt_loss_weight` with same types